README.md

```diff
@@ -144,6 +144,23 @@ especially in the case in which the exporter is external to the code it's monito
 
 For more detailed documentation on [Availability exporters](https://gitlab.cee.redhat.com/konflux/docs/documentation/-/blob/main/o11y/monitoring/availability_exporters.md?ref_type=heads)
 
+## Recording Rules
+
+Recording rules allow us to precompute frequently needed or computationally expensive expressions
+and save their result as a new set of time series. Recording rules are the go-to approach for
+speeding up the performance of queries that take too long to return. When other teams want to go
+with their own metrics format for exporters they need to adapt to desired metric form by
+translating it using a recording rule. 
+
+These recording rules should be put in the [rhobs/recording folder](https://github.com/redhat-appstudio/o11y/tree/main/rhobs). 
+
+Below is the metric we need from recording rules
+ ```
+ grafana_ds_up(check=prometheus-appstudio-ds) -> konflux_up(service=grafana, check=prometheus-appstudio-ds)
```

Comment:
We cannot anticipate the format of the original metric, so we need to provide clear requirements for the availability metric that we want to have eventually. I.e.

- Which labels the metric should have?
- What should those labels represent?
- What value of the metrics means up and what means down?
- We should require the metric to provide value both when it's up and when it's down (we know of metrics that only report when up - they are hard to work with).

We can mention a supporting example, but the requirements should be clear regardless.

Reply:
Updated